dtreeviz/trees.py

```diff
@@ -1334,6 +1345,43 @@ def _draw_piechart(counts, size, colors, filename, label=None, fontname="Arial",
     plt.close()
 
 
+def _draw_barh_chart(counts, size, colors, filename, label=None, fontname="Arial", ticks_fontsize=9, graph_colors=None):
+    graph_colors = adjust_colors(graph_colors)
+    n_nonzero = np.count_nonzero(counts)
+
+    if n_nonzero != 0:
+        i = np.nonzero(counts)[0][0]
+        if n_nonzero == 1:
+            counts = [counts[i]]
+            colors = [colors[i]]
+
+    tweak = size * .01
+    fig, ax = plt.subplots(1, 1, figsize=(size, 0.3))
```

Comment:
This looks amazing. The bar charts look a little thick to me. I wonder if 0.2 would look better?